jbmc/src/java_bytecode/library/.gitignore

```diff
@@ -0,0 +1,5 @@
+src
+classes
+converter
+core-models.jar
+java_core_models.inc
```

Comment:
We don't do per-directory gitignore files elsewhere, so this might cause confusion. I'd suggest to include this in the top-level .gitignore file.